ui/components/app/token-cell/token-cell.test.js

```diff
@@ -23,6 +40,28 @@ describe('Token Cell', () => {
     },
   };
 
+  // two tokens with the same symbol but different addresses
+  const MOCK_GET_TOKEN_LIST = {
+    '0xAnotherToken': {
+      name: 'TEST',
+      erc20: true,
+      symbol: 'TEST',
+      decimals: 18,
+      address: '0xAnotherToken',
+      iconUrl: './images/test_image.svg',
+      aggregators: [],
+    },
+    '0xAddress': {
+      name: 'TEST-2',
+      erc20: true,
+      symbol: 'TEST',
+      decimals: 18,
+      address: '0xAddress',
+      iconUrl: './images/test_1_image.svg',
+      aggregators: [],
+    },
```

Comment:
I think the order of these tokens should be swapped.  That way we're testing it skips over the first symbol match, and lands on the correct symbol + address match.  Otherwise this test passes on the old code, because it picks the first one which happens to be correct.